app/enterprise/0.34-x/installation/docker.md

```diff
@@ -9,12 +9,12 @@ title: Installing Kong Enterprise Docker Image
 A guide to installing Kong Enterprise—and its license file—using
 Docker. **Trial users should skip directly to step 4**.
 
-1. Log in to [bintray.com](https://bintray.com). Your credentials will have 
+1. Log in to <a href="https://bintray.com" target="_blank">bintray.com</a>. Your credentials will have 
 been emailed to you by your Sales or Support contact.
```

Comment:
Rework this sentence, sounds awkward

Reply:
Oof, future perfect tense/passive voice--how about this:

"Your Sales or Support contact will have emailed your credentials to you."